addons/project/static/src/components/project_control_panel/project_control_panel.js

```diff
@@ -10,8 +10,8 @@ export class ProjectControlPanel extends ControlPanel {
         super.setup();
         this.orm = useService("orm");
         this.user = useService("user");
-        const { active_id, show_project_update } = this.env.searchModel.globalContext;
-        this.showProjectUpdate = this.env.config.viewType === "form" || show_project_update;
+        const { active_model, active_id, show_project_update } = this.env.searchModel.globalContext;
+        this.showProjectUpdate = (this.env.config.viewType === "form" || show_project_update) && active_model === "project.project";
```

Comment:
What else than project could it be? :thinking:

Reply:
It is due to a `active_model` problem. Xavier explained it like this : 
"when the user clicks on search more on a many2one field pointing on `project.task`, if the active_model and the active_id are defined then if we will try to load the last update status by using `active_id` even if `active_model` is not the`project.project` model."

Reply:
Ok thanks for the explanation :fire:  FW-JS team made a fix in this area. Could please check again if needed?